tests/pytest_tests/unit_tests/test_wandb_save.py

```diff
@@ -1,79 +1,163 @@
-import os
-import tempfile
+import pathlib
 
 import pytest
-from wandb.sdk.lib import filesystem
 
 # ----------------------------------
 # wandb.save
 # ----------------------------------
 
 
-@pytest.mark.xfail(reason="This test is flaky")
-def test_save_policy_symlink(mock_run, parse_records, record_q):
+def test_save_relative_path(
+    monkeypatch,
+    tmp_path: pathlib.Path,
+    mock_run,
+    parse_records,
+    record_q,
+):
+    # Use a fake working directory for the test.
+    monkeypatch.chdir(tmp_path)
```

Comment:
i'm not sure it is required, we have this auto fixture in conftest (https://github.com/wandb/wandb/blob/main/tests/pytest_tests/conftest.py#L53), maybe we can redo it so it is more explicit, but no need to change.

Reply:
interesting... I'll think about this